src/app/modules/main/wallet_section/send/io_interface.nim

```diff
@@ -14,7 +14,7 @@ method load*(self: AccessInterface) {.base.} =
 method isLoaded*(self: AccessInterface): bool {.base.} =
   raise newException(ValueError, "No implementation available")
 
-method deleteAccount*(self: AccessInterface, keyUid: string, address: string) {.base.} =
+method deleteAccount*(self: AccessInterface, address: string) {.base.} =
```

Comment:
pretty sure this function can be remove all together as you didn't change anything else?

Reply:
Exactly, didn't check that it has no implementation in its module. Removed.